### PR TITLE
Broadcast transactions, in bulk

### PR DESCRIPTION
**Pull request by NiftyWell**
Updated NetworkProvider interface and added a method to send transactions in bulk.

### CODE REVIEW

File: src/apiNetworkProvider.ts [hunk 1]
```diff
@@ -120,6 +120,15 @@ export class ApiNetworkProvider implements INetworkProvider {
         return response.txHash;
     }
 
+    async sendBulkTransaction(txs: ITransaction[]): Promise<string> {
```
Comment (andreibancioiu) [inline src/apiNetworkProvider.ts:123]: Perhaps the name `sendTransactions()` is a bit more suitable.

File: src/apiNetworkProvider.ts [hunk 2]
```diff
@@ -120,6 +120,15 @@ export class ApiNetworkProvider implements INetworkProvider {
...
+        let sendable: any = [];
```
Comment (andreibancioiu) [inline src/apiNetworkProvider.ts:124]: Variables can be declared as `const`.

File: src/apiNetworkProvider.ts [hunk 3]
```diff
@@ -120,6 +120,15 @@ export class ApiNetworkProvider implements INetworkProvider {
...
+        txs.forEach((tx) => {
```
Comment (andreibancioiu) [inline src/apiNetworkProvider.ts:125]: Alternatively, `map()` can be used, perhaps.

File: src/apiNetworkProvider.ts (hunk 1 above)
Comment (andreibancioiu) [reply src/apiNetworkProvider.ts:123]: Also, perhaps implement this as `this.backingProxyNetworkProvider.sendTransactions()`.

**Review by andreibancioiu — APPROVED** [review]
@NiftyWell, thank you for your contribution :tada:  

We will merge it into the branch `external-12-17`, then apply a few adjustments to prepare the release.

:pray:

**Review by schimih — APPROVED** [review]

**Pull request merged**